concordia/templates/base.html

```diff
@@ -56,7 +56,7 @@ <h1 class="header-text m-0 d-none d-sm-flex align-items-center">
                             <a class="nav-link {% if PATH_LEVEL_1 == 'about'%}active{% endif %}" href="{% url 'about' %}">About</a>
                         </li>
                         <li class="nav-item">
-                            <a class="nav-link {% if PATH_LEVEL_1 == 'campaigns' %}active{% endif %}" href="{% url 'transcriptions:campaign-list' %}">Campaigns</a>
+                            <a class="nav-link {% if PATH_LEVEL_1 == 'campaigns' %}active{% endif %}" href="{% url 'campaign-topic-list' %}">Campaigns</a>
```

Comment:
Are we launching this immediately without a feature-gate?

Reply:
Yes